app/page.tsx

```diff
@@ -1,4 +1,8 @@
+import AuthButton from '../components/AuthButton';
```

Comment:
Revert these file changes. Keep your Commit clean from the thing only things that should change.